src/main/java/org/checkerframework/checker/noliteral/qual/MaybeConstant.java

```diff
@@ -0,0 +1,36 @@
+package org.checkerframework.checker.noliteral.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.DefaultInUncheckedCodeFor;
+import org.checkerframework.framework.qual.LiteralKind;
+import org.checkerframework.framework.qual.QualifierForLiterals;
+import org.checkerframework.framework.qual.SubtypeOf;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+/**
+ * The top type for the NoLiteral Checker's type system.
```

Comment:
Start by saying what it means.  This fact is obvious from the annotations and is not the most important information that a reader of the Javadoc will want to know.